### PR TITLE
yadage: moved specification loading logic to reana-commons

### PR DESCRIPTION
**Pull request by audrium**
closes https://github.com/reanahub/reana-server/issues/358

Depends on:
- https://github.com/reanahub/pytest-reana/pull/83
- https://github.com/reanahub/reana-commons/pull/264

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
     'cwl-utils==0.5 ; python_version>="3"',
     "pyOpenSSL>=19.0.0",  # FIXME remove once yadage-schemas solves deps.
     "jsonpointer>=2.0",
-    "reana-commons>=0.8.0a13,<0.9.0",
+    "reana-commons[yadage]>=0.8.0a13,<0.9.0",
```
Comment (mvidalgarcia) [inline setup.py:50]: Should we remove `yadage-schemas` dep?

I also see many deps that likely depend on `yadage-schemas` (# FIXME remove once yadage-schemas solves deps.), should we move those to reana-commons? Or maybe give it a try if those dep issues are solved.

Comment (audrium) [reply setup.py:49]: Nicely spotted! Indeed after some testing it seems only `reana-commons[yadage]` is enough

**Review by mvidalgarcia — COMMENTED** [review]
It needs rebase.

Edit: Rebased.

**Review by mvidalgarcia — APPROVED** [review]
Tests passing installing `reana-commons[yadage]` and `pytest-reana` locally ✅

**Comment by mvidalgarcia** [comment]
Changelog please.

**Review by ParthS007 — APPROVED** [review]
LGTM :)

**Pull request merged**